lib/matplotlib/colorbar.py

```diff
@@ -805,7 +805,7 @@ def add_lines(self, *args, **kwargs):
         xy = self.ax.transAxes.inverted().transform(inches.transform(xy))
         col.set_clip_path(mpath.Path(xy, closed=True),
                           self.ax.transAxes)
-        self.ax.add_collection(col)
+        self.ax.add_collection(col, autolim=False)
```

Comment:
Here as well: These are decorations on the colorbar and should not influence its limits.